README.md

```diff
@@ -23,28 +23,10 @@ python setup.py install
 
 Once the installation is complete, ICGC get can be invoked with the command `icgc-get`
 
-ICGC get is also available as a docker container.  Running ICGC get through a 
-docker container will prevent issues from arising related to conflicting 
-software requirements for the data download clients. The docker container will also
-automatically install all supported data clients.
-
-First, install docker from https://docs.docker.com/mac/. Then pull the docker image using the command
-
-```shell
-docker pull icgc/icgc-get
-```
-
-To make working with the container easier, it is recommend to add a bash alias to enable invocation of ICGC get with the command
-`icgc-get`.
-
-```shell
-alias icgc-get="docker run -it --rm -v $MOUNT_DIR:/icgc/mnt icgc/icgc-get --config /icgc/mnt/config.yaml"
-```
-
-replacing `$MOUNT_DIR` with the path to your mounted directory. By default, this directory will be populated by the script with
-process logs and downloaded files. You will also need to save a configuration file in this directory if you wish
-to pass a customized config file to the container. The files will be written with ownership set to the current user (`/usr/bin/id -u`) and group (`/usr/bin/id -g`).
-
+If you do not have any of download clients installed locally, ICGC get is capable of running them through docker. 
+Running any of the clients through a docker container will prevent issues from arising related to conflicting 
+software requirements for the data download clients.  To enable this functionality, first install 
+docker from https://docs.docker.com/mac/.
```

Comment:
I think this should be a markdown hyperlink, and the target should be be platform agnostic. Also, please capitalize Docker unless it is obvious you mean the command `docker`.